editor/plugins/texture_region_editor_plugin.cpp

```diff
@@ -826,6 +826,7 @@ void TextureRegionEditor::_notification(int p_what) {
 
 		case NOTIFICATION_READY: {
 			panner->setup((ViewPanner::ControlScheme)EDITOR_GET("editors/panning/sub_editors_panning_scheme").operator int(), ED_GET_SHORTCUT("canvas_item_editor/pan_view"), bool(EDITOR_GET("editors/panning/simple_panning")));
+			panner->set_viewport(get_viewport());
```

Comment:
Originally, I placed the new code side-by-side to `setup`, but I have noticed that here it is in `NOTIFICATION_READY`. Would it be better, to move `setup` and `set_viewport` to `NOTIFICATION_ENTER_TREE`?

Reply:
It can be moved.